.github/workflows/nintendo.yaml

```diff
@@ -0,0 +1,40 @@
+# based on https://angband.readthedocs.io/en/latest/hacking/compiling.html#nintendo-ds-nintendo-3ds
+name: Nintendo DS
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  _3ds:
+    name: Nintendo 3DS
+    runs-on: ubuntu-latest
+    container: devkitpro/devkitarm
+    steps:
+      - name: Clone Project
+        uses: actions/checkout@v2
+
+      - name: Build
+        shell: bash
+        run: |
+          pushd src/
+          make -f Makefile.3ds
+          popd
+          test -e src/angband.3dsx
```

Comment:
Do the other targets hardcode `angband`? It will most likely be worth it to look into how that is configured, so that all the derivatives don't have to change all their GitHub actions manually.

Reply:
Reconfigured output file https://github.com/angband/angband/pull/5309/commits/0019bdea0699acce58fc3873a2670dfc6727e9c8

Reply:
This location here is still using the hardcoded binary name, as does the one below.